pkg/kds/context/context.go

```diff
@@ -43,11 +44,11 @@ var log = core.Log.WithName("kds")
 
 type Context struct {
 	ZoneClientCtx         context.Context
+	TypesSentByZone       []core_model.ResourceType
+	TypesSentByGlobal     []core_model.ResourceType
 	GlobalProvidedFilter  reconcile_v2.ResourceFilter
 	ZoneProvidedFilter    reconcile_v2.ResourceFilter
 	GlobalServerFiltersV2 []mux.FilterV2
-	// Configs contains the names of system.ConfigResource that will be transferred from Global to Zone
-	Configs map[string]bool
```

Comment:
This field is updated in KM, not sure if you can remove it

Reply:
Let me look.

Reply:
Ok changed this to be a global var. This feels less akward than the accidental sharing that is happening now.

Reply:
There will be more KM changes needed anyway with the change to the flags